src/frames/UtilFrame.java

```diff
@@ -537,7 +543,7 @@ private void jCheckBox9ActionPerformed() {//GEN-FIRST:event_jButton4ActionPerfor
         ArrayVisualizer.toggleExternalArrays(jCheckBox9.isSelected());
     }//GEN-LAST:event_jCheckBox8ActionPerformed
 
-    private void jComboBox1ActionPerformed() {//GEN-FIRST:event_jButton4ActionPerformed
+    private void jComboBox1ActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_jButton4ActionPerformed
```

Comment:
What's the usage of adding this parameter?

Reply:
I have now removed the extraneous parameter.